docs/cli.md

```diff
@@ -291,6 +311,7 @@ sb run [--config-file]
        [--no-docker]
        [--output-dir]
        [--private-key]
+       [--get-info]
```

Comment:
pls keep the list sorted, put before host-file line

Reply:
thanks, updated